astropy/modeling/tests/test_core.py

```diff
@@ -558,7 +558,7 @@ def test_rename_path(tmpdir):
     paths = [os.path.dirname(astropy.__path__[0])] + sys.path
     env['PYTHONPATH'] = os.pathsep.join(paths)
 
-    script = tmpdir.join('rename.py').strpath
+    script = tmp_path / 'rename.py'
     with open(script, 'w') as f:
         f.write(MODEL_RENAME_CODE)
```

Comment:
It is possible to shorten the code here:
```suggestion
    (script := tmp_path / "rename.py").write_text(MODEL_RENAME_CODE)
```

Reply:
I'm not a huge fan of this change.